data/doc_pages/en/installation.md

```diff
@@ -12,15 +12,86 @@ If it is not the case, please refer to :
 
 - Docker Compose : [docs.docker.com/compose/overview/](https://docs.docker.com/compose/overview/)
 
+- mkvirtualenvwrapper : [https://virtualenvwrapper.readthedocs.io/en/latest/index.html](https://virtualenvwrapper.readthedocs.io/en/latest/index.html)
+
 
 ## Installation
 
+### Odoo backend installation
+
+#### Simple Installation
+To simply try out Shopinvader, here are the lines to copy in a requirement.txt file:
+```
+--find-links https://wheelhouse.acsone.eu/manylinux1
+--extra-index https://wheelhouse.shopinvader.com/simple/
+--extra-index https://wheelhouse.odoo-community.org/oca-simple/
+
+-r https://raw.githubusercontent.com/odoo/odoo/12.0/requirements.txt
+https://nightly.odoo.com/12.0/nightly/src/odoo_12.0.latest.zip
+
+https://github.com/OCA/openupgradelib/archive/master.tar.gz
+
+odoo12_addon_shopinvader_demo_app
+
+```
+
+Other available payment methods can be added from shopinvader-payment such as:
+
+```
+odoo12-addon-shopinvader-payment-stripe
+odoo12-addon-shopinvader-payment-sips
+```
+
+In a virtualenv(<strong>running python3</strong>) install Shopinvader using the requirement.txt file with:
+
+```
+pip install -r ./requirements.txt
```

Comment:
`python -m pip install...` is the recommanded way to run pip to be sure people use the same pip as the virtualenv.